sdk/storage/azure-storage-files-shares/inc/azure/storage/files/shares/protocol/share_rest_client.hpp

```diff
@@ -5828,7 +5836,12 @@ namespace Azure { namespace Storage { namespace Files { namespace Shares {
           Azure::Core::Nullable<std::string>
               FileCacheControl; // Sets the file's cache control. The File service stores this value
                                 // but does not use or modify it.
-          Azure::Core::Nullable<std::string> FileContentMd5; // Sets the file's MD5 hash.
+          Azure::Core::Nullable<Storage::ContentHash>
+              ContentMd5; // An MD5 hash of the content. This hash is used to verify the integrity
```

Comment:
Is this transactional or not?

It should be named `TransactionalContentHash` if it's transactional, otherwise, `ContentHash`.

Reply:
ditto for the others.

Reply:
The variable name is not visible to customers.